.prod/uwsgi.ini

```diff
@@ -9,3 +9,9 @@ gid = appuser
 master = 1
 processes = 2
 threads = 2
+cron = -1 -1 -1 -1 -1 /app/manage.py send_mail
+cron = 0 -1 -1 -1 -1 /app/manage.py retry_deferred
```

Comment:
so, you want to run this every 20 minutes?
```
cron = -20 -1 -1 -1 -1 /app/manage.py retry_deferred
```

Reply:
yeb, the idea is to retry deferred emails every 20 mins

Reply:
ok! you could use the line I suggested instead of the three you have 🙂

Reply:
Ahhh. Yes! 😆Damn, why used three lines for this